### DISCUSSION TITLE
Is there a way to view the entire log?

### DISCUSSION THREAD

**Discussion by ykdojo** [vercel/community#1816] [Help] [answered]
### Summary

As the attached screenshot shows, when a log exceeds a certain length (56+ lines in this case), it seems to be cut off at the beginning.

Is there a way to view the entire log?

![image](https://user-images.githubusercontent.com/1811651/226155622-f542ee53-0666-4f5d-8d68-a439fafc53e2.png)

**Comment by swarnava** [comment, marked as answer]
Hi, this is the current limitation of Serverless Function. We get log from “tail log” of lambda invocation that limited to 4KB including metadata it contains. We remove these metadata before display the log so that only the last 3.81KB of the log message is shown.
